web-server/src/components/AnimatedInputWrapper/AnimatedInputWrapper.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { themeColors } from "@/theme/schemes/theme";
+
+const AnimatedInputWrapper = () => {
+  const [isAnimationVisible, setIsAnimationVisible] = useState(true);
+  const [value, setValue] = useState("");
+
+  const addAnimation = useCallback(
+    () => setIsAnimationVisible(true),
+    [setIsAnimationVisible]
+  );
+
+  const handleChange = useCallback(
+    (e: any) => setValue(e.target.value),
```

Comment:
Don't use any unless you absolutely have to.
In this case, you can use the ChangeEvent type or something similar.

Reply:
I have removed this as it was redundant